src/reducers/userReducer.js

```diff
@@ -0,0 +1,12 @@
+export const currentUserState = {
+  currentUserID: "5f06550d61b7a5b8c0afb334",
```

Comment:
should this be set after we retrieve the user after authentication?

Reply:
Definitely, I branched out before you added your login flow stuff so I haven't updated this part yet, it's hardcoded as a particular user for now.